flight/PiOS/inc/pios_can.h

```diff
@@ -35,6 +35,7 @@
 
 //! The set of CAN messages
 enum pios_can_messages {
+	PIOS_CAN_ERROR = -1,
```

Comment:
Yeah if you want to do that it would be great. I think what I wrote should work, there might be one other place using that check is required. Then I'll verify CAN still works since it is a small functionality change.